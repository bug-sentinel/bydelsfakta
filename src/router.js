import Vue from 'vue';
import Router from 'vue-router';
import Home from './views/Home.vue';
import Bydel from './views/Bydel.vue';
import Alder from './views/subviews/Alder.vue';
import Boligpriser from './views/subviews/Boligpriser.vue';
import Bygningstyper from './views/subviews/Bygningstyper.vue';
import Eierform from './views/subviews/Eierform.vue';
import Flytting from './views/subviews/Flytting.vue';
import Folkemengde from './views/subviews/Folkemengde.vue';
import Husholdninger from './views/subviews/Husholdninger.vue';
import Innvandrerbefolkningen from './views/subviews/Innvandrerbefolkningen.vue';
import Kommunaleboliger from './views/subviews/Kommunaleboliger.vue';
import Levekaar from './views/subviews/Levekaar.vue';
import Trangboddhet from './views/subviews/Trangboddhet.vue';

import Graph from './components/Graph.vue';

Vue.use(Router);

export default new Router({
  routes: [
    {
      path: '/',
      name: 'Home',
      component: Home,
    },
    {
      path: '/graphA',
      name: 'Graph1',
      props: { settings: { url: '/mockDataA.json', template: 'a' } },
      component: Graph,
    },
    {
      path: '/graphB',
      name: 'Graph2',
      props: { settings: { url: '/mockDataA2.json', template: 'a' } },
      component: Graph,
    },
    {
      path: '/graphC',
      name: 'Graph3',
      props: { settings: { url: '/mockDataB.json', template: 'b' } },
      component: Graph,
    },
    {
      path: '/bydel/:bydel',
      name: 'Bydel',
      component: Bydel,
    },
    {
      path: '/bydel/:bydel/folkemengde',
      name: 'BydelFolkemengde',
      component: Folkemengde,
    },
    {
      path: '/bydel/:bydel/alder',
      name: 'BydelAlder',
      component: Alder,
    },
    {
      path: '/bydel/:bydel/innvandrerbefolkningen',
      name: 'BydelInnvandrerbefolkningen',
      component: Innvandrerbefolkningen,
    },
    {
      path: '/bydel/:bydel/husholdninger',
      name: 'BydelHusholdninger',
      component: Husholdninger,
    },
    {
      path: '/bydel/:bydel/trangboddhet',
      name: 'BydelTrandboddhet',
      component: Trangboddhet,
    },
    {
      path: '/bydel/:bydel/eierform',
      name: 'BydelEierform',
      component: Eierform,
    },
    {
      path: '/bydel/:bydel/kommunaleboliger',
      name: 'BydelKommunaleboliger',
      component: Kommunaleboliger,
    },
    {
      path: '/bydel/:bydel/flytting',
      name: 'BydelFlytting',
      component: Flytting,
    },
    {
      path: '/bydel/:bydel/bygningstyper',
      name: 'BydelBygningstyper',
      component: Bygningstyper,
    },
    {
      path: '/bydel/:bydel/boligpriser',
      name: 'BydelBoligpriser',
      component: Boligpriser,
    },
    {
      path: '/bydel/:bydel/levekaar',
      name: 'Bydellevekaar',
      component: Levekaar,
    },
    {
      path: '/sammenlign/:bydel*',
      name: 'Sammenlign',
      component: Bydel,
      children: [
        {
          path: '/folkemengde',
          name: 'Folkemengde',
          component: Folkemengde,
        },
        {
          path: '/alder',
          name: 'Alder',
          component: Alder,
        },
        {
          path: '/innvandrerbefolkningen',
          name: 'Innvandrerbefolkningen',
          component: Innvandrerbefolkningen,
        },
        {
          path: '/husholdninger',
          name: 'Husholdninger',
          component: Husholdninger,
        },
        {
          path: '/trangboddhet',
          name: 'Trandboddhet',
          component: Trangboddhet,
        },
        {
          path: '/eierform',
          name: 'Eierform',
          component: Eierform,
        },
        {
          path: '/kommunaleboliger',
          name: 'Kommunaleboliger',
          component: Kommunaleboliger,
        },
        {
          path: '/flytting',
          name: 'Flytting',
          component: Flytting,
        },
        {
          path: '/bygningstyper',
          name: 'Bygningstyper',
          component: Bygningstyper,
        },
        {
          path: '/boligpriser',
          name: 'Boligpriser',
          component: Boligpriser,
        },
        {
          path: '/levekaar',
          name: 'levekaar',
          component: Levekaar,
        },
      ],
    },
  ],
});
