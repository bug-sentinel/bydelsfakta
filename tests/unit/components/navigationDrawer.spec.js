import { mount, createLocalVue } from '@vue/test-utils';
import NavigationDrawer from '../../../src/components/NavigationDrawer.vue';
import router from '../../../src/router';

describe('NavigationDrawer', () => {
  let wrapper = null;

  beforeEach(() => {
    const localVue = createLocalVue();
    localVue.use(router);
    router.push('/bydel/sagene');
    wrapper = mount(NavigationDrawer, {
      localVue,
      router,
    });
  });

  afterEach(() => {
    wrapper.destroy();
  });

  test('renders navigationDrawer-component and finds navbar-id', () => {
    expect(wrapper.attributes().id).toBe('navbar');
  });

  test('renders navigationDrawer correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  test('render navigationDrawer with route /bydel/gamleoslo correctly', () => {
    router.push('/bydel/gamleoslo');
    expect(wrapper.element).toMatchSnapshot();
  });

  test('render navigationDrawer with route /bydel/1-2-3-4 correctly', () => {
    router.push('/bydel/1-2-3-4');
    expect(wrapper.vm.$route.path).toEqual('/bydel/1-2-3-4');
    router.push('/bydel/test');
    expect(wrapper.vm.$route.path).toEqual('/bydel/test');
  });

  test('change selectedPredefinedOption to length of 3', () => {
    expect(wrapper.vm.selected.length).toEqual(1);
    wrapper.vm.selectedPredefinedOption = ['1', '2', '3'];
    expect(wrapper.vm.selected.length).toEqual(3);
  });

  test('selectedPredefinedOption is length of 0', () => {
    wrapper.vm.selectedPredefinedOption = [];
    expect(wrapper.vm.selectedPredefinedOption.length).toEqual(0);
  });

  test('selectedPredefinedOption is set to length of 3 and router is pushed to Tema', () => {
    router.push('/bydel/sagene/alder');
    wrapper.vm.selectedPredefinedOption = ['1', '2', '3'];
    expect(wrapper.vm.selected).toEqual(['1', '2', '3']);
    expect(wrapper.vm.$route.path).toEqual('/bydel/1-2-3/alder');
  });
});

describe('created lifecycle hook', () => {
  test('params is alle', () => {
    const localVue = createLocalVue();
    localVue.use(router);
    router.push('/bydel/alle');
    const wrapper = mount(NavigationDrawer, {
      localVue,
      router,
    });

    expect(wrapper.element).toMatchSnapshot();
  });

  test('params.length is bigger than 0', () => {
    const localVue = createLocalVue();
    localVue.use(router);
    router.push('/bydel/1-2-3-4');
    const wrapper = mount(NavigationDrawer, {
      localVue,
      router,
    });

    expect(wrapper.vm.compareBydeler).toEqual(true);
    expect(wrapper.element).toMatchSnapshot();
  });

  test('params is undefined', () => {
    const localVue = createLocalVue();
    localVue.use(router);
    router.push('/bydel/');
    const wrapper = mount(NavigationDrawer, {
      localVue,
      router,
    });

    expect(wrapper.element).toMatchSnapshot();
  });

  test('bydel is 1 instead of sagene', () => {
    const localVue = createLocalVue();
    localVue.use(router);
    router.push('/bydel/1');
    const wrapper = mount(NavigationDrawer, {
      localVue,
      router,
    });

    expect(wrapper.element).toMatchSnapshot();
  });

  test('bydel is sagene and tema is alder', () => {
    const localVue = createLocalVue();
    localVue.use(router);
    router.push('/bydel/sagene/alder');
    const wrapper = mount(NavigationDrawer, {
      localVue,
      router,
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
