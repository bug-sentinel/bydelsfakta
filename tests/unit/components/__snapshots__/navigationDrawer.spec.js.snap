// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheNavigationDrawer render TheNavigationDrawer with route /bydel/gamleoslo correctly 1`] = `
<aside
  class="navbar"
>
  <a
    class="logo-link router-link-active"
    href="/"
  >
    <img
      alt="oslo-logo"
      class="oslo__logo"
      src=""
    />
  </a>
   
  <button
    class="selectedTopic"
  >
    Sammenlign bydeler (1)
  </button>
   
  <div
    class="version-tag"
  >
    Alpha
  </div>
   
  <nav
    class="navigation"
    role="navigation"
  >
    <ul
      class="navigation-list"
    >
      <li
        class="navigation-link navigation-link--compare"
      >
        <input
          id="checkbox-gamleoslo"
          type="checkbox"
          value="01"
        />
         
        <label
          class="compare"
          for="checkbox-gamleoslo"
        />
         
        <a
          class="navigation-link__label router-link-exact-active router-link-active"
          href="/bydel/gamleoslo"
          id="a-gamleoslo"
        >
          Bydel Gamle Oslo
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-grunerlokka"
          type="checkbox"
          value="02"
        />
         
        <label
          class="compare"
          for="checkbox-grunerlokka"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/grunerlokka"
          id="a-grunerlokka"
        >
          Bydel Grünerløkka
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-sagene"
          type="checkbox"
          value="03"
        />
         
        <label
          class="compare"
          for="checkbox-sagene"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/sagene"
          id="a-sagene"
        >
          Bydel Sagene
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-sthanshaugen"
          type="checkbox"
          value="04"
        />
         
        <label
          class="compare"
          for="checkbox-sthanshaugen"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/sthanshaugen"
          id="a-sthanshaugen"
        >
          Bydel St. Hanshaugen
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-frogner"
          type="checkbox"
          value="05"
        />
         
        <label
          class="compare"
          for="checkbox-frogner"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/frogner"
          id="a-frogner"
        >
          Bydel Frogner
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-ullern"
          type="checkbox"
          value="06"
        />
         
        <label
          class="compare"
          for="checkbox-ullern"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/ullern"
          id="a-ullern"
        >
          Bydel Ullern
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-vestreaker"
          type="checkbox"
          value="07"
        />
         
        <label
          class="compare"
          for="checkbox-vestreaker"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/vestreaker"
          id="a-vestreaker"
        >
          Bydel Vestre Aker
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-nordreaker"
          type="checkbox"
          value="08"
        />
         
        <label
          class="compare"
          for="checkbox-nordreaker"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/nordreaker"
          id="a-nordreaker"
        >
          Bydel Nordre Aker
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-bjerke"
          type="checkbox"
          value="09"
        />
         
        <label
          class="compare"
          for="checkbox-bjerke"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/bjerke"
          id="a-bjerke"
        >
          Bydel Bjerke
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-grorud"
          type="checkbox"
          value="10"
        />
         
        <label
          class="compare"
          for="checkbox-grorud"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/grorud"
          id="a-grorud"
        >
          Bydel Grorud
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-stovner"
          type="checkbox"
          value="11"
        />
         
        <label
          class="compare"
          for="checkbox-stovner"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/stovner"
          id="a-stovner"
        >
          Bydel Stovner
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-alna"
          type="checkbox"
          value="12"
        />
         
        <label
          class="compare"
          for="checkbox-alna"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/alna"
          id="a-alna"
        >
          Bydel Alna
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-ostensjo"
          type="checkbox"
          value="13"
        />
         
        <label
          class="compare"
          for="checkbox-ostensjo"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/ostensjo"
          id="a-ostensjo"
        >
          Bydel Østensjø
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-nordstrand"
          type="checkbox"
          value="14"
        />
         
        <label
          class="compare"
          for="checkbox-nordstrand"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/nordstrand"
          id="a-nordstrand"
        >
          Bydel Nordstrand
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-sondrenordstrand"
          type="checkbox"
          value="15"
        />
         
        <label
          class="compare"
          for="checkbox-sondrenordstrand"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/sondrenordstrand"
          id="a-sondrenordstrand"
        >
          Bydel Søndre Nordstrand
        </a>
      </li>
       
      <li
        class="navigation-link navigation-link__label-compare navigation-link--active"
        id="sammenlign"
      >
        <a
          class="navigation-link__label"
          href="/bydel/alle"
          id="sammenlign-href"
        >
          Sammenlign bydeler
        </a>
      </li>
    </ul>
     
    <div
      class="navigation-drawer__buttons"
      name="fade"
    >
      <div
        class="navigation-drawer__button-container"
      >
        <button
          aria-label="velg alle checkboxes"
          class="navigation-drawer__button"
        >
          
            Velg alle
          
        </button>
         
        <button
          aria-label="fjern alle checkboxes"
          class="navigation-drawer__button"
        >
          
            Fjern alle
          
        </button>
      </div>
       
      <div
        class="navigation-drawer__select-container"
      >
        <label
          class="visually-hidden"
          for="navigation-drawer-select"
        >
          Velg region
        </label>
         
        <select
          class="navigation-drawer__select"
          id="navigation-drawer-select"
        >
          <option
            disabled="disabled"
            value=""
          >
            - Velg region -
          </option>
          <option
            value="04,05,06,07,08"
          >
            Vest
          </option>
          <option
            value="01,02,03,09,10,11,12,13,14,15"
          >
            Øst
          </option>
          <option
            value="01,02,03,04,05"
          >
            Indre by
          </option>
          <option
            value="01,02,03"
          >
            Indre by øst
          </option>
          <option
            value="04,05"
          >
            Indre by vest
          </option>
          <option
            value="06,07,08,09,10,11,12,13,14,15"
          >
            Ytre by
          </option>
          <option
            value="06,07,08"
          >
            Ytre by vest
          </option>
          <option
            value="09,10,11,12"
          >
            Ytre by øst
          </option>
          <option
            value="13,14,15"
          >
            Ytre by sør
          </option>
          <option
            value="01,02,03,04,05,06,07,08,09,10,11,12,13,14,15"
          >
            Velg alle
          </option>
        </select>
      </div>
    </div>
  </nav>
</aside>
`;

exports[`TheNavigationDrawer renders TheNavigationDrawer correctly 1`] = `
<aside
  class="navbar"
>
  <a
    class="logo-link router-link-active"
    href="/"
  >
    <img
      alt="oslo-logo"
      class="oslo__logo"
      src=""
    />
  </a>
   
  <button
    class="selectedTopic"
  >
    Sammenlign bydeler (15)
  </button>
   
  <div
    class="version-tag"
  >
    Alpha
  </div>
   
  <nav
    class="navigation"
    role="navigation"
  >
    <ul
      class="navigation-list"
    >
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-gamleoslo"
          type="checkbox"
          value="01"
        />
         
        <label
          class="compare"
          for="checkbox-gamleoslo"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/gamleoslo"
          id="a-gamleoslo"
        >
          Bydel Gamle Oslo
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-grunerlokka"
          type="checkbox"
          value="02"
        />
         
        <label
          class="compare"
          for="checkbox-grunerlokka"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/grunerlokka"
          id="a-grunerlokka"
        >
          Bydel Grünerløkka
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-sagene"
          type="checkbox"
          value="03"
        />
         
        <label
          class="compare"
          for="checkbox-sagene"
        />
         
        <a
          class="navigation-link__label router-link-exact-active router-link-active"
          href="/bydel/sagene"
          id="a-sagene"
        >
          Bydel Sagene
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-sthanshaugen"
          type="checkbox"
          value="04"
        />
         
        <label
          class="compare"
          for="checkbox-sthanshaugen"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/sthanshaugen"
          id="a-sthanshaugen"
        >
          Bydel St. Hanshaugen
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-frogner"
          type="checkbox"
          value="05"
        />
         
        <label
          class="compare"
          for="checkbox-frogner"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/frogner"
          id="a-frogner"
        >
          Bydel Frogner
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-ullern"
          type="checkbox"
          value="06"
        />
         
        <label
          class="compare"
          for="checkbox-ullern"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/ullern"
          id="a-ullern"
        >
          Bydel Ullern
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-vestreaker"
          type="checkbox"
          value="07"
        />
         
        <label
          class="compare"
          for="checkbox-vestreaker"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/vestreaker"
          id="a-vestreaker"
        >
          Bydel Vestre Aker
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-nordreaker"
          type="checkbox"
          value="08"
        />
         
        <label
          class="compare"
          for="checkbox-nordreaker"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/nordreaker"
          id="a-nordreaker"
        >
          Bydel Nordre Aker
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-bjerke"
          type="checkbox"
          value="09"
        />
         
        <label
          class="compare"
          for="checkbox-bjerke"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/bjerke"
          id="a-bjerke"
        >
          Bydel Bjerke
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-grorud"
          type="checkbox"
          value="10"
        />
         
        <label
          class="compare"
          for="checkbox-grorud"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/grorud"
          id="a-grorud"
        >
          Bydel Grorud
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-stovner"
          type="checkbox"
          value="11"
        />
         
        <label
          class="compare"
          for="checkbox-stovner"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/stovner"
          id="a-stovner"
        >
          Bydel Stovner
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-alna"
          type="checkbox"
          value="12"
        />
         
        <label
          class="compare"
          for="checkbox-alna"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/alna"
          id="a-alna"
        >
          Bydel Alna
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-ostensjo"
          type="checkbox"
          value="13"
        />
         
        <label
          class="compare"
          for="checkbox-ostensjo"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/ostensjo"
          id="a-ostensjo"
        >
          Bydel Østensjø
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-nordstrand"
          type="checkbox"
          value="14"
        />
         
        <label
          class="compare"
          for="checkbox-nordstrand"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/nordstrand"
          id="a-nordstrand"
        >
          Bydel Nordstrand
        </a>
      </li>
      <li
        class="navigation-link"
      >
        <input
          id="checkbox-sondrenordstrand"
          type="checkbox"
          value="15"
        />
         
        <label
          class="compare"
          for="checkbox-sondrenordstrand"
        />
         
        <a
          class="navigation-link__label"
          href="/bydel/sondrenordstrand"
          id="a-sondrenordstrand"
        >
          Bydel Søndre Nordstrand
        </a>
      </li>
       
      <li
        class="navigation-link navigation-link__label-compare navigation-link--active"
        id="sammenlign"
      >
        <a
          class="navigation-link__label router-link-exact-active router-link-active"
          href="/bydel/sagene"
          id="sammenlign-href"
        >
          Sammenlign bydeler
        </a>
      </li>
    </ul>
     
    <div
      class="navigation-drawer__buttons"
      name="fade"
    >
      <div
        class="navigation-drawer__button-container"
      >
        <button
          aria-label="velg alle checkboxes"
          class="navigation-drawer__button"
        >
          
            Velg alle
          
        </button>
         
        <button
          aria-label="fjern alle checkboxes"
          class="navigation-drawer__button"
          disabled="disabled"
        >
          
            Fjern alle
          
        </button>
      </div>
       
      <div
        class="navigation-drawer__select-container"
      >
        <label
          class="visually-hidden"
          for="navigation-drawer-select"
        >
          Velg region
        </label>
         
        <select
          class="navigation-drawer__select"
          id="navigation-drawer-select"
        >
          <option
            disabled="disabled"
            value=""
          >
            - Velg region -
          </option>
          <option
            value="04,05,06,07,08"
          >
            Vest
          </option>
          <option
            value="01,02,03,09,10,11,12,13,14,15"
          >
            Øst
          </option>
          <option
            value="01,02,03,04,05"
          >
            Indre by
          </option>
          <option
            value="01,02,03"
          >
            Indre by øst
          </option>
          <option
            value="04,05"
          >
            Indre by vest
          </option>
          <option
            value="06,07,08,09,10,11,12,13,14,15"
          >
            Ytre by
          </option>
          <option
            value="06,07,08"
          >
            Ytre by vest
          </option>
          <option
            value="09,10,11,12"
          >
            Ytre by øst
          </option>
          <option
            value="13,14,15"
          >
            Ytre by sør
          </option>
          <option
            value="01,02,03,04,05,06,07,08,09,10,11,12,13,14,15"
          >
            Velg alle
          </option>
        </select>
      </div>
    </div>
  </nav>
</aside>
`;
